menus/linux.cson

```diff
@@ -125,6 +125,10 @@
         ]
       }
       { type: 'separator' }
+      { label: '&Increase Font Size', command: 'window:increase-font-size' }
+      { label: '&Decrease Font Size', command: 'window:decrease-font-size' }
+      { label: '&Reset Font Size', command: 'window:reset-font-size' }
```

Comment:
Will this cause problems with this command in the same menu:

```
       { label: '&Reload', command: 'window:reload' }
```

We would now have two menu options starting with `&R...`